terraform/variables.tf

```diff
@@ -173,3 +179,27 @@ variable "distributed_tff_example_worker_2_address" {
   description = "Address of the second worker of the distributed TensorFlow Federated example. Set this when the worker is outside the coordinator mesh."
   type        = string
 }
+
+variable "cross-device" {
```

Comment:
Nit: rename to `cross_device` (note the `_`).

Remember to update all the references.

Reply:
I still see the old definition